plugin.rb

```diff
@@ -82,7 +82,15 @@ module ::DiscourseDataExplorer
         field :query_id, component: :choices, required: true, extra: { content: queries }
         field :query_params, component: :"key-value", accepts_placeholders: true
         field :skip_empty, component: :boolean
-        field :attach_csv, component: :boolean
+        field :attach_csv,
+              component: :boolean,
+              validator: ->(attach_csv) do
+                if attach_csv && !SiteSetting.authorized_extensions.split("|").include?("csv")
```

Comment:
I think we also need to handle the "*" which allows for "all" extensions.

Reply:
oh didn't know we do this...

Reply:
@ZogStriP ok I added the * case thanks